src/server/protocol.d.ts

```diff
@@ -126,6 +126,14 @@ declare module ts.server.protocol {
     }
 
     /**
+      * Go to type request; value of command field is
+      * "type". Return response giving the file locations that

```

Comment:
I thought about that... but everything else was short.. 
"gototype", why not "gotodefinition" then
"typedefintion", long, but i like it better.